app/assets/stylesheets/application.css.scss

```diff
@@ -5,3 +5,10 @@
 @import 'identity-style-guide/dist/assets/scss/styles';
 @import 'components/all';
 @import 'print';
+
+// This import is temporary and scoped to the new Document Capture screen. The File Input component
+// is part of a USWDS version newer than what's currently shipped with Login Design System. Follow-
+// up work should upgrade to use a newer USWDS, at which time this should be removed altogether.
+#document-capture-form {
+  @import 'uswds/dist/scss/elements/form-controls/file-input';
+}
```

Comment:
In some last-minute testing to see what potential side-effects could occur from this import, I noticed that the USWDS file input stylesheet applies some styles to [file inputs globally](https://github.com/uswds/uswds/blob/705c2b2f7db8c0ae6ce013c0697173dbf237edb1/src/stylesheets/elements/form-controls/_file-input.scss#L1-L6). While the version of USWDS which is bundled with Login Design System [does include some as well](https://www.runpkg.com/?identity-style-guide@2.1.5/dist/assets/scss/uswds/elements/_inputs.scss#323), they are not the same (margin, padding). This has the result that file inputs elsewhere in the application would have their styling altered unintentionally.

As a workaround, I updated the `@import` to be scoped to [the `#document-capture-form` element](https://github.com/18F/identity-idp/blob/035e88843c5be9f2c277329a6dcda44cf16d7558/app/views/idv/doc_auth/document_capture.html.erb#L12) used as the mounting point for the new React application, to ensure that styling only takes effect for the new page, until the design system can be upgraded as a whole.